src/api/video.ts

```diff
@@ -6,10 +6,14 @@ export interface GetVideoTokenProps {
   roomName: string;
 }
 
+export interface VideoTokenResponse {
+  data: { token: string };
+}
+
 export const getVideoToken = async ({
   userId,
   roomName,
-}: GetVideoTokenProps): Promise<void | firebase.functions.HttpsCallableResult> => {
+}: GetVideoTokenProps): Promise<void | VideoTokenResponse> => {
```

Comment:
See comments I made in the previous review thread RE: transforming this data so it only returns the token itself, not the 'boxed return shape' (that is an implementation detail of the backend function/etc, not a good 'public API' to expose)

Reply:
fixed in prev thrread